translations/de/module_de_lang.php

```diff
@@ -15,13 +15,22 @@
     'wd_account_holder_title' => 'Kontoinhaber',
     'wd_amount' => 'Betrag',
     'wd_bic' => 'BIC',
+    'wd_birthdate_format_php_code' => 'd.m.Y',
+    'wd_birthdate_format_user_hint' => 'DD.MM.JJJJ',
```

Comment:
Maybe `wd_date_format_placeholder`?

Reply:
I changed it to wd_date_format_user_hint